packages/compass-saved-aggregations-queries/src/components/select-connection-and-namespace-modal.tsx

```diff
@@ -0,0 +1,377 @@
+import React, { useCallback, useMemo } from 'react';
+import {
+  Checkbox,
+  FormModal,
+  Option,
+  Select,
+  css,
+  cx,
+  spacing,
+} from '@mongodb-js/compass-components';
+import {
+  ColorCircleGlyph,
+  useConnectionColor,
+} from '@mongodb-js/connection-form';
+import { connect } from 'react-redux';
+import type { MapDispatchToProps, MapStateToProps } from 'react-redux';
+import type { RootState } from '../stores';
+import {
+  type State as OpenItemState,
+  closeModal,
+  connectionSelected,
+  openSelectedItem,
+  selectCollection,
+  selectDatabase,
```

Comment:
can we have consistent naming for the 3 select events? no preference on which one :)

Reply:
yea sorry for being lazy - i am sure that I just jump past that part 🙈 
fixed this in [6190a0a](https://github.com/mongodb-js/compass/pull/5841/commits/6190a0a66ed1005abc26258666f38c0a23a98c60)